install

```diff
@@ -53,7 +53,7 @@ def main():
         # (from a local dir if it exists)
         # "python -m pip install" instead of "pip install" handles a pip
         # issue where it fails in a long-named dir
-        run('virtualenv --python=python3 venv && '
+        run('virtualenv --python=python3.7 venv && '
```

Comment:
since we also support manof atop python 3.9, this needs to stay `python3` and below, on project.toml, leave it to be either 3.7 or 3.9

Reply:
@liranbg  - thanks for the review!
Getting back to this now. I understand manof needs compatibility with both 3.7 and 3.9, but since this is an install script, several questions:

1. developers - I suppose they should all align to 3.9, right? there should be a single developer version for the repo. even if it's overridable, or BC with other python version. traditionally this is the highest supported version. Should we change CI scripts to install and test 3.9?
2. Is there a scenario where install script is used sometimes in 3.7.X env and sometimes in 3.9.x env? I think it's only used in the makefile for dev environment. but maybe in the internal iguazio deployment script it is called from somewhere as well so that's why it should support both?

Reply:
soft ping @liranbg

Reply:
Hi @omesser  (❤️ )
1. CI may install and test python 3.9 indeed (or even better, matrix of 3.7/3.9)
2. Indeed we use it with 3.7 on internals and 3.9 for deployments

Reply:
have some free time this week (I hope) I'll issue a PR to try and get this CI to run on a matrix of python versions. should be simple